README.md

```diff
@@ -80,6 +80,7 @@ Note: Using `nvm` (or [fnm](https://github.com/Schniz/fnm)), run `nvm install --
 Create a `.env` which should override environment variables required to run locally:
 ```properties
 HMPPS_AUTH_URL=http://localhost:9090/auth
+MANAGE_USERS_API_URL=http://localhost:9091/manage-users-api
```

Comment:
HMPPS Auth endpoint, `/auth/api/user/me` is deprecated. This info now comes from the manage users API, (`/users/me`)